go.sh

```diff
@@ -0,0 +1,7 @@
+./riak_test -c riak_ee_dev -t repl_fs_bench3 -D'fs_num_keys=1000000' -D'fs_strategy=keylist' -D'fs_direct=0'
```

Comment:
@javajolt I rebased the riak_test sweeper branch and got rid of those two commits, and everything else still seems to work. Just pushed the updated branch.